### PR TITLE
Add boss analagous to glossary/guide but for boss mechanics

### PR DESCRIPTION
**Pull request by RheingoldRiver**

### CODE REVIEW

**Comment by RheingoldRiver** [comment]
I think for like a month or 2 we can add temporary regular padglobal commands for each of the current existing boss mechanics commands that say to use `^boss reiwa` etc instead and then after that delete the old ones

File: padglobal/padglobal.py [hunk 1]
```diff
@@ -1092,6 +1159,10 @@ def setGeneral(self, cmd):
         self.clearCmd(cmd)
         self.save_settings()
 
+    def setBoss(self, cmd):
```
Comment (nachoapps) [inline padglobal/padglobal.py]: This is unnecessary (and unused).

File: padglobal/padglobal.py [hunk 2]
```diff
@@ -572,6 +575,69 @@ def lookup_glossary(self, term):
...
+        op = 'EDITED' if term in self.settings.boss() else 'ADDED'
+        self.settings.addBoss(term, definition)
+        await self.bot.say("PAD boss mechanics successfully {}.".format(op))
+
+    @padglobal.command(pass_context=True)
+    async def rmboss(self, ctx, *, term):
+        """Adds a set of boss mechanics."""
+        term = term.lower()
+        if term not in self.settings.boss():
+            await self.bot.say("Glossary item doesn't exist.")
```
Comment (nachoapps) [inline padglobal/padglobal.py]: 'glossary item'

Comment (RheingoldRiver) [reply padglobal/padglobal.py]: ^speling omg fixed

File: padglobal/padglobal.py (hunk 1 above)
Comment (RheingoldRiver) [reply padglobal/padglobal.py]: fixed

**Pull request merged**